docs/api.md

```diff
@@ -755,7 +755,7 @@ await articles.sync({ignoreBackoff: true});
 
 ### `Kinto` instance
 
-Using the `events` on a `Kinto` instance property you can subscribe public events from. That `events` property implements nodejs' [EventEmitter interface](https://nodejs.org/api/events.html#events_class_events_eventemitter).
+Using the `events` property you can subscribe to public events from a `Kinto` instance.   That `events` property implements nodejs' [EventEmitter interface](https://nodejs.org/api/events.html#events_class_events_eventemitter).
```

Comment:
Thanks for catching this! I've tweaked it just a smidge.

```suggestion
Using the `events` property you can subscribe to public events from a `Kinto` instance. The `events` property implements the [EventEmitter interface](https://nodejs.org/api/events.html#events_class_eventemitter).
```

Reply:
I also kicked around The/This/That for the second sentence, but didn't manage to decide... Your tweak sounds good. Cheers